public/js/issuer.js

```diff
@@ -14,18 +14,6 @@ document.on("DOMContentLoaded", async e => {
     "/.well-known/trust-token/key-commitment"
   )).json();
 
-  // display chrome flag with key-commitment
-  const CMD = `/Applications/Google\\ Chrome\\ Canary.app/Contents/MacOS/Google\\ Chrome\\ Canary \\
-  --additional-trust-token-key-commitments='${JSON.stringify(COMMITMENT)}' \\
```

Comment:
if key pair of server updated, key commitment will change too.
so it should be generated dynamically from here. please don't remove.

Reply:
Got it! I'll change this.